spec/controllers/application_controller_spec.rb

```diff
@@ -185,4 +185,14 @@ def index
       end
     end
   end
+
+  describe '#create_user_event' do
+    before { sign_in_as_user }
+
+    it 'creates an Event object' do
+      expect(Event).to receive(:create)
+
+      subject.create_user_event(:account_created)
+    end
+  end
```

Comment:
This can be rewritten so that it doesn't use the DB, such as:
```ruby
describe '#create_user_event' do
  context 'when the user is not specified' do
    it 'creates an Event object for the current_user' do
      user = build_stubbed(:user)

      allow(subject).to receive(:current_user).and_return(user)

      expect(Event).to receive(:create).
        with(user_id: user.id, event_type: :account_created)

      subject.create_user_event(:account_created)
    end
  end

  context 'when the user is specified' do
    it 'creates an Event object for the specified user' do
      user = build_stubbed(:user)

      expect(Event).to receive(:create).
        with(user_id: user.id, event_type: :account_created)

      subject.create_user_event(:account_created, user)
    end
  end
end
```